forge-gui/res/cardsfolder/upcoming/craig_boone_novac_guard.txt

```diff
@@ -0,0 +1,14 @@
+Name:Craig Boone, Novac Guard
+ManaCost:1 R W
+Types:Legendary Creature Human Soldier
+PT:3/3
+K:Reach
+K:Lifelink
+Reach, lifelink
+T:Mode$ AttackersDeclared | Execute$ TrigPutCounter | ValidAttackers$ Creature | ValidAttackersAmount$ GE2 | TriggerZones$ Battlefield | AttackingPlayer$ You | TriggerDescription$ One for My Baby — Whenever you attack with two or more creatures, put two quest counters on CARDNAME. When you do, NICKNAME deals damage equal to the number of quest counters on it to up to one target creature unless that creature's controller has CARDNAME deal that much damage to them.
+SVar:TrigPutCounter:DB$ PutCounter | CounterType$ QUEST | CounterNum$ 2 | RememberPut$ True  | SubAbility$ DBImmediateTrigger 
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Card | Execute$ TrigDealDamage | SpellDescription$ When you do, NICKNAME deals damage equal to the number of quest counters on it to up to one target creature unless that creature's controller has CARDNAME deal that much damage to them.
+SVar:TrigDealDamage:DB$ DealDamage | ValidTgts$ Creature | TgtPrompt$ Select up to one target creature | TargetMin$ 0 | TargetMax$ 1 | NumDmg$ X | UnlessCost$ DamageYou<X> | UnlessPayer$ TargetedController
```

Comment:
Seems like it recognizes the correct amount (with the prompt saying do you want to deal X to targeted) but it doesn't seem to deal damage on resolution

Reply:
works for me?

Reply:
You're right actually